core/templates/installation.php

```diff
@@ -49,7 +49,8 @@
 				value="<?php p($_['adminpass']); ?>"
 				autocomplete="off" autocapitalize="none" autocorrect="off" required>
 			<label for="adminpass" class="infield"><?php p($l->t( 'Password' )); ?></label>
-			<input type="checkbox" id="show" class="hidden-visually" name="show">
+			<input type="checkbox" id="show" class="hidden-visually" name="show"
+				   aria-label="Show password">
```

Comment:
Please use a translation method :)
```suggestion
				   aria-label="<?php p($l->t( 'Show password' )); ?>">
```